src/components/tooltip/Tooltip.tsx

```diff
@@ -48,21 +42,23 @@ const TooltipContainer = styled.div`
 const TooltipClose = styled.span`
   cursor: pointer;
 `;
-
-export interface TooltipProps
-  extends Omit<React.HTMLProps<HTMLDivElement>, 'as' | 'ref'> {
+export interface TooltipProps extends Omit<React.HTMLProps<HTMLDivElement>, 'as' | 'ref'> {
+  /** Clickable text to display that fires onHintClose */
   closeLabel?: string;
   /** Text or Element to display in the tooltip */
   description: string | JSX.Element;
+  /** If omitted, you need to control visible state to show/hide the tooltip */
   displayTrigger?: 'click' | 'hover';
   /** CSS ID */
   id?: string;
-  /** Function to call on close action */
+  /** Function to call when clicking on closeLabel */
   onHintClose?: () => void;
   placement: 'top' | 'bottom' | 'left' | 'right';
   type?: 'hint' | 'tooltip';
   /** if true, the tooltip should be displayed */
   visible?: boolean;
+  /** Optional CSS class name for wrapping span element  */
```

Comment:
I see in the code that we use `div` l.144

Reply:
👍 fixing.